package.json

```diff
@@ -1,25 +1,27 @@
 {
   "name": "beslutteroversikt",
   "version": "0.1.0",
+  "type": "module",
   "private": true,
   "homepage": "/beslutteroversikt",
   "scripts": {
-    "start": "cross-env REACT_APP_DEPLOYMENT_ENV=local craco start",
-    "build:dev": "cross-env REACT_APP_DEPLOYMENT_ENV=development PUBLIC_URL=/ craco build --config craco.config.js",
-    "build:prod": "cross-env REACT_APP_DEPLOYMENT_ENV=production PUBLIC_URL=/ craco build --config craco.config.js",
-    "build:mock": "cross-env REACT_APP_DEPLOYMENT_ENV=local craco build",
-    "test": "echo \"Tests not yet implemented\"",
+    "start": "vite",
+    "build:prod": "tsc && vite build",
+    "build:dev": "tsc && vite build",
```

Comment:
Jeg tror ikke `build:dev` bruker riktig `.env`-fil nå (dvs. `.env.development`).
Ref. [Env Variables and Modes - Vite](https://vitejs.dev/guide/env-and-mode#modes):

> By default, the dev server (dev command) runs in development mode and the build command runs in production mode.
>
> This means when running vite build, it will load the env variables from .env.production if there is one:

Reply:
Den skal ikke bruke den heller, `.env.development` er til lokal utvikling, men å kalle den `.local` kunne være risky siden noen i fremtiden kan tenke at den ikke pushes til git og derfor legger noe hemmelig i den 🙃
Men det er kanskje et litt vel stort forbehold å ta, så endrer den tilbake til `.local` med kommentar i filen.